src/txn/txn_recover.c

```diff
@@ -989,6 +994,18 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
         __wt_err(session, ret, "Recovery failed");
     }
 
+    /*
+     * Close all the cursors and open dhandles if rollback to stable was called to make sure that
+     * all the modified pages as part of RTS are evicted from the cache.
+     *
+     */
+    if (rollback_to_stable_called) {
+        F_SET(conn, WT_CONN_CLOSING);
```

Comment:
Add a comment to explain why we use the CLOSING flag also?

Reply:
Updated comments